src/AKCore/Scripts/VueComponents/Upcoming/UpcomingListItem.vue

```diff
@@ -31,6 +31,9 @@
         <p v-if="event.startsTime && event.type === 'Spelning'">
           {{ t("concert-starts", "common") }}: {{ event.startsTime }}
         </p>
+        <p v-if="event.playDuration && event.type === 'Spelning'">
```

Comment:
Perhaps this should also be added to Scripts\VueComponents\Upcoming\EventInfoModal.vue
for the calendar view:
![image](https://github.com/LudHag/AKCore/assets/8194231/06a00758-faae-45d1-ad1e-0f7c2de64d38)